### PR TITLE
Added implementation for creating x report

### PR DESCRIPTION
**Pull request by StefanKert**
This PR introduces the change for extending the Zero Receipt to support creation of x-reports

#298

### CODE REVIEW

File: scu-it/src/fiskaltrust.Middleware.SCU.IT.EpsonRTPrinter/EpsonRTPrinterSCU.cs [hunk 1]
```diff
@@ -594,16 +593,32 @@ private async Task<ReceiptResponse> PerformDailyCosing(ReceiptResponse receiptRe
...
+        if (request.IsXReportZeroReceipt())
+        {
+            var fiscalReport = new FiscalReport
+            {
+                XReport = new XReport()
+            };
+            var response = await SendRequestAsync(SoapSerializer.Serialize(fiscalReport));
+            using var responseContent = await response.Content.ReadAsStreamAsync();
+            var reportResponse = SoapSerializer.DeserializeToSoapEnvelope<ReportResponse>(responseContent);
+            if (!result?.Success ?? false)
```
Comment (volllly) [inline scu-it/src/fiskaltrust.Middleware.SCU.IT.EpsonRTPrinter/EpsonRTPrinterSCU.cs:610]:
```suggestion
            if (!(result?.Success ?? false))
```
Just makes it more clear what's going on with out having to think about operator precedence ^^

File: scu-it/test/fiskaltrust.Middleware.SCU.IT.AcceptanceTests/EpsonRTPrinterTests.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@ namespace fiskaltrust.Middleware.SCU.IT.AcceptanceTests
 {
     public class EpsonRTPrinterTests : ITSSCDTests
     {
-        private static readonly Uri _serverUri = new Uri("http://192.168.0.14/");
+        private static readonly Uri _serverUri = new Uri("http://194.93.177.143:9000");
```
Comment (volllly) [inline scu-it/test/fiskaltrust.Middleware.SCU.IT.AcceptanceTests/EpsonRTPrinterTests.cs:9]: IDK if you want to share the IP address 🤷

Comment (StefanKert) [reply scu-it/test/fiskaltrust.Middleware.SCU.IT.AcceptanceTests/EpsonRTPrinterTests.cs:9]: Ehm .. no thanks good catch :D 

```suggestion
        private static readonly Uri _serverUri = new Uri("http://192.168.0.14/");
```

File: scu-it/src/fiskaltrust.Middleware.SCU.IT.EpsonRTPrinter/version.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.3.53",
+  "version": "1.3.60-rc1",
```
Comment (StefanKert) [inline scu-it/src/fiskaltrust.Middleware.SCU.IT.EpsonRTPrinter/version.json:2]:
```suggestion
  "version": "1.3.60",
```

File: scu-it/test/fiskaltrust.Middleware.SCU.IT.AcceptanceTests/ReceiptRequests/PosReceipts/0x0001_Cash_WithNegativeAmount.json [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      "CostCenter": "4",
+      "MoneyGroup": "1",
+      "MoneyNumber": ""
+    }
+  ],
+  "ftReceiptCase": 5283883447318740993,
+  "cbReceiptAmount": 2.80,
+  "cbUser": "Chef",
+  "cbArea": "53"
+}
```
Comment (StefanKert) [inline scu-it/test/fiskaltrust.Middleware.SCU.IT.AcceptanceTests/ReceiptRequests/PosReceipts/0x0001_Cash_WithNegativeAmount.json]:
```suggestion
{
  "ftCashBoxID": "bb734f2c-24fa-4ec8-9346-b2893cb7a98e",
  "ftQueueID": "497981fb-e60c-4e6c-b058-b6cb46a7e144",
  "ftPosSystemId": "497981fb-e60c-4e6c-b058-b6cb46a7e144",
  "cbTerminalID": "1",
  "cbReceiptReference": "497981fb-e60c-4e6c-b058-b6cb46a7e144",
  "cbReceiptMoment": "2024-01-09T10:03:22.7Z",
  "cbChargeItems": [
    {
      "Position": 100,
      "Quantity": 1.0000,
      "Description": "Felsenkeller",
      "Amount": 4.80000000000000000000000000,
      "VATRate": 10.0000,
      "ftChargeItemCase": 5283883447184523281,
      "ftChargeItemCaseData": "",
      "VATAmount": 0.4363636363636363636363636364,
      "CostCenter": "4",
      "ProductGroup": "Biere",
      "ProductNumber": "5009",
      "ProductBarcode": "",
      "Unit": "Stk",
      "Moment": "2024-05-09T10:03:11.233Z"
    },
    {

      "Position": 200,
      "Quantity": 1.0000,
      "Description": "Klein",
      "Amount": -2.00000000000000000000000000,
      "VATRate": 10.0000,
      "ftChargeItemCase": 5283883447184523281,
      "ftChargeItemCaseData": "",
      "VATAmount": -0.1818181818181818181818181818,
      "CostCenter": "4",
      "ProductGroup": "Pizza",
      "ProductNumber": "40041",
      "ProductBarcode": "",
      "Unit": "Stk",
      "Moment": "2024-05-09T10:03:17.173Z"
    }
  ],
  "cbPayItems": [
    {
      "Quantity": 1.0,
      "Description": "Bar",
      "Amount": 2.8000,
      "ftPayItemCase": 5283883447184523265,
      "ftPayItemCaseData": "",
      "CostCenter": "4",
      "MoneyGroup": "1",
      "MoneyNumber": ""
    }
  ],
  "ftReceiptCase": 5283883447318740993,
  "cbReceiptAmount": 2.80,
  "cbUser": "Chef",
  "cbArea": "1"
}
```

**Review by volllly — APPROVED** [review]

**Pull request merged**